docs/develop/auto_sdk/auto-xdm.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Auto XDM
+sidebar_position: 4
+description: Functions to utilize Auto XDM (Cross-Domain Messaging)
+keywords:
+    - Developers Documentation
+    - DevDocs
+    - SDK
+    - Auto SDK
+    - Auto XDM
+---
+
+### Cross-Domain Transfers
+
+- `transfer(api: ApiPromise, destination: ChainOrDomain, receiver: string, amount: Amount): Promise<SubmittableExtrinsic>`: Creates a transaction to transfer a specified amount to a receiver on a given destination, which can be either the consensus chain or a specific domain.
+
+#### Types:
+
+- **Amount**: `BigInt | number | string`  
+  Represents the amount to be transferred.
+
+- **Consensus**: `{ type: 'consensus' }`  
+  Indicates that the destination is the consensus chain.
+
+- **Domain**: `{ type: 'domain'; domainId: number }`  
+  Specifies a domain as the destination, identified by a `domainId`.
+
+- **ChainOrDomain**: `Consensus | Domain`  
+  A union type representing either the consensus chain or a domain.
+
+---
+
+**Usage Example:**
```

Comment:
Larger header would be good here imo. The usage example is what readers will look for first.